README.md

```diff
@@ -116,6 +116,11 @@ Clone this repository to your desired folder:
 - Twitter: [@twitterhandle](https://twitter.com/josuerushanika)
 - LinkedIn: [LinkedIn](https://www.linkedin.com/in/josue-rushanika-336049b3/)
 
+## 🚀 Live Demo <a name="live-demo"></a>
```

Comment:
Well done on adding the demo. Kindly, try to include features. it would be better you delete anything in your table of content that does not have content.